deployments/fargate/README.md

```diff
@@ -0,0 +1,121 @@
+# AWS Fargate Deployment
+Familiarity with AWS Fargate (Fargate) is assumed. Consult the 
+[User Guide for AWS Fargate](https://docs.aws.amazon.com/AmazonECS/latest/userguide/what-is-fargate.html) for further reading.
+
+Unless stated otherwise, it is assumed that the [Splunk OpenTelemetry Collector](https://github.com/signalfx/splunk-otel-collector)
+(Collector) is deployed to an ECS Task in a **sidecar** container alongside monitored applications.
+
+Applies to Collector release v0.30.0 and above. See the Docker image repository
+[here](https://quay.io/repository/signalfx/splunk-otel-collector?tab=tags).
+
+
+## Default Configuration
+The default configuration file is located at `/etc/otel/collector/fargate_config.yaml`
+in the Collector image. See 
+[here](https://github.com/signalfx/splunk-otel-collector/blob/main/cmd/otelcol/config/collector/fargate_config.yaml)
+for its contents and 
+[here](https://github.com/signalfx/splunk-otel-collector/blob/main/cmd/otelcol/Dockerfile)
+for the Collector image Dockerfile. Note that the default metrics receivers are `hostmetrics`
+and `smartagent/ecs-metadata`.
+
+Default configuration steps in the container definition for the Collector:
+- Map the endpoint ports in the default configuration file (i.e. `13133`, `6060`,
+  `55679`, `14250`, `6832`, `6831`, `14268`, `8888`, `7276`, `9943`, `9411`, `9080`).
+- Assign the default configuration file path to environment variable `SPLUNK_CONFIG`.
+- Assign a list of metrics you want excluded to environment variable `METRICS_TO_EXCLUDE`.
+- Assign a list of images whose metrics you want excluded to environment variable `IMAGES_TO_EXCLUDE`.
+
+## Custom Configuration
+For the Collector pick up your custom configuration, you need to assign the path to your custom
+configuration file to environment variable `SPLUNK_CONFIG` in the container definition for
```

Comment:
Isn't `SPLUNK_CONFIG_YAML` the only option to provide a custom config at the moment? With `SPLUNK_CONFIG` user can define only another existing pre-built config which are not custom but specific to other use-cases (agent, gateway). So we probably should put instructions from _Direct Configuration_ section here?

Reply:
No, `SPLUNK_CONFIG` can be used to specify non-default config file.

Reply:
Sorry I don't understand. `SPLUNK_CONFIG` stores a file path to a config, right? If user cannot mount any external volume to the container, how can they provide a custom config?

Reply:
Users can mount volumes to tasks in Fargate. The mounted volume(s) are accessible to all containers in task. `SPLUNK_CONFIG_YAM` is just for convenience.